src/main/java/sh/mob/timer/web/Room.java

```diff
@@ -24,18 +24,41 @@ final class Room {
   private final List<TimerRequest> timerRequests = new CopyOnWriteArrayList<>();
   private final Sinks.Many<TimerRequest> sink =
       Sinks.many().replay().latestOrDefault(NULL_TIMER_REQUEST);
+  private Goal currentGoal = Goal.NO_GOAL;
+
+  private final Sinks.Many<Goal> goalRequestSink =
+      Sinks.many().replay().latestOrDefault(Goal.NO_GOAL);
```

Comment:
I'm new to this reactive style. Do we need both currentGoal and the goalRequestSink to maintain this state?
It feels strange having to keep both in sync.

Reply:
Yes we need these. The Goal is the internal State where the sink ist just for the sever sent events.